### ISSUE TITLE
Clicking your link gives error

### ISSUE THREAD

**Issue by jamminjames** [StartingAndroid/FirebaseCloudMessaging#1]
Clicking your "Android Push Notification Using Firebase Cloud Messaging" link gives this error:
Non-Image content-type returned

**Comment by rajazeeshanahmed** [comment]
Yes, because I didn't renew the hosting so it expires and all the contents get deleted. Sorry
